subiquity/common/types.py

```diff
@@ -732,3 +732,16 @@ class CasperMd5Results(enum.Enum):
     FAIL = 'fail'
     PASS = 'pass'
     SKIP = 'skip'
+
+
+class MirrorCheckStatus(enum.Enum):
+    OK = enum.auto()
+    RUNNING = enum.auto()
+    FAILED = enum.auto()
```

Comment:
fwiw I think something more explicit than enum.auto() is a good idea here (esp if we ever expect the desktop installer to call these apis!)

Reply:
oh yeah, definitely!